scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -111,6 +115,33 @@ class FossId(
 
             return projectName
         }
+
+        /**
+         * This function fetches credentials for [repoUrl] and insert them between the URL scheme and the host. If no
+         * matching host is found by [OrtAuthenticator], the [repoUrl] is returned untouched.
+         */
+        private fun queryOrtAuthenticator(repoUrl: String): String {
```

Comment:
The function name (and docs) still talk about "OrtAuthenticator" whereas you do not directly query the `OrtAuthenticator` anymore, so we should just say "Authenticator".